apps/nextjs/src/components/board/sections/gridstack/use-gridstack.ts

```
@@ -53,44 +84,89 @@ export const useGridstack = ({ section, mainRef }: UseGridstackProps): UseGrista
 
   const onChange = useCallback(
     (changedNode: GridStackNode) => {
-      const itemId = changedNode.el?.getAttribute("data-id");
-      if (!itemId) return;
-
-      // Updates the react-query state
-      moveAndResizeItem({
-        itemId,
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        xOffset: changedNode.x!,
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        yOffset: changedNode.y!,
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        width: changedNode.w!,
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        height: changedNode.h!,
-      });
+      const id = changedNode.el?.getAttribute("data-id");
+      const type = changedNode.el?.getAttribute("data-type");
+
+      if (!id || !type) return;
+
+      if (type === "item") {
+        // Updates the react-query state
+        moveAndResizeItem({
+          itemId: id,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          xOffset: changedNode.x!,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          yOffset: changedNode.y!,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          width: changedNode.w!,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          height: changedNode.h!,
+        });
```

Explain with a comment why you suppress the error